tests/test_dojo.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+from classes.dojo import Dojo
+
+
```

Comment:
Have a `class setUp` that holds the repetitive actions like creating instance of Dojo()